app/controllers/persons_controller.rb

```diff
@@ -8,6 +8,7 @@ class PersonsController < ApplicationController
 
   def index
     @persons = Person.all
+    @fcn_link = Page.find_by_title("Frequently called numbers")
```

Comment:
@cdoyle-temple Now that we have slugs on this model, do you want to update this page with a slug and query by slug instead?

Reply:
I was going to save all that until these mini pages are done, and then go back and fill them all in in. But it's your call. Let me know.